Makefile.nt

```diff
@@ -0,0 +1,49 @@
+# Makefile for Windows NT using Watcom compiler.
+#
+# wmake -f Makefile.nt
+
+CC=wcc386
+CPPFLAGS=-DHAVE_SIGNAL_H -DHAVE_SYS_TIME_H -DWIN32
```

Comment:
No sys/time.h in ow19, and it builds fine without sys/time.h: please remove -DHAVE_SYS_TIME_H flag.

Reply:
OK, done.